README.md

```diff
@@ -155,8 +155,8 @@ python3 -m pip install torch numpy sentencepiece
 # convert the 7B model to ggml FP16 format
 python3 convert-pth-to-ggml.py models/7B/ 1
 
-# quantize the model to 4-bits
-python3 quantize.py 7B
+# quantize the model to 4-bits (using method 2 = q4_0)
+./quanize models/7B/ggml-model-f16.bin ./models/7B/ggml-model-q4_0.bin 2
```

Comment:
```suggestion
./quantize models/7B/ggml-model-f16.bin ./models/7B/ggml-model-q4_0.bin 2
```

Reply:
Fixed in f4c4d29d7274e8fc4381f973f3be61a30a85e17c

(i really should go to sleep 😄 )